cli/list/list.go

```diff
@@ -46,3 +50,129 @@ func ListInstances(cmdCtx *cmdcontext.CmdCtx, cliOpts *config.CliOpts) error {
 
 	return nil
 }
+
+// printBinaries outputs installed versions of the program.
+func printBinaries(program string, binList []string) {
+	if len(binList) > 0 {
+		log.Infof(program + ":")
+		for _, bin := range binList {
+			if strings.HasSuffix(bin, "active") {
+				fmt.Printf("	%s\n", util.Bold(color.GreenString(bin)))
+			} else {
+				fmt.Printf("	%s\n", color.YellowString(bin))
+			}
+		}
+	}
+}
+
+// sortBinaryVersions sorts versions of binary.
+func sortBinaryVersions(binList []string) {
+	sort.SliceStable(binList, func(i, j int) bool {
+		if binList[i] == "master" {
+			return true
+		}
+
+		if binList[j] == "master" {
+			return false
+		}
+
+		varLeftString := strings.Split(binList[i], ".")
+		varRightString := strings.Split(binList[j], ".")
+		var right []uint64
+		var left []uint64
+
+		for _, ch := range varLeftString {
+			i, _ := strconv.Atoi(ch)
+			left = append(left, uint64(i))
+		}
+
+		for _, ch := range varRightString {
+			i, _ := strconv.Atoi(ch)
+			right = append(right, uint64(i))
+		}
+
+		largestLen := util.Max(len(left), len(right))
```

Comment:
As I can see, this code repeats what was done in `SortVersions` function in `version` package. Please don't duplicate the code. Versions comparator must be part of the `version` package. For example version.Compare of ver1.LessThan(ver2) and must be used by the code which works with versions comparison and in SortVersions function.

Reply:
changed code, now use version.SortVersions